apps/web/src/pages/activities/services/chart-bar/option.service.ts

```diff
@@ -10,6 +10,14 @@ export function getOptions(this: any, isTriggerSent: boolean, daysCount: number)
       display: false,
     },
 
+    onHover: (event, el) => {
+      if (el.length > 0) {
+        event.native.target.style.cursor = 'pointer';
+      } else {
+        event.native.target.style.cursor = 'default';
+      }
+    },
```

Comment:
this is a hover trick, basically, the chart is a canvas, so there is no easy way of saying that on the bar hover it should show the cursor pointer, but we have this callback which is fired when hovering over the chart and if there are elements we do change the cursor style